addons/web/static/src/views/form/form_controller.scss

```diff
@@ -73,6 +73,11 @@ $o-form-label-margin-right: 0px;
         min-height: $font-size-base * $line-height-base;
     }
 
+    // Explicitly set elements as full-width
+    .o_form_content_fullwidth {
+        @include o-form-nosheet-negative-margin();
```

Comment:
Hello @Brieuc-brd , I have to add this class for the accounting app (not urgent).
Since you're refactoring the view you may take this need in account, maybe we can find a cleaner way  ;)